README.md

```diff
@@ -9,3 +9,11 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Let $G_1 = (V_1, E_1)$ and $G_2 = (V_2, E_2)$, and let both graphs have n nodes and be completely connected.  Being completely connected, this means there is an edge between every pair of nodes, allowing for a one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u, v) \in E_1$ iff $(f(u), f(v)) \in E_2$.  A bijection is possible, as there is always a way for a node to be connected to another node no matter the graph.  There are no isolated nodes, and a corresponding node can always be found in the other graph.
```

Comment:
But how do you know this? How can you find a corresponding node in the other graph for any node?